Source/objects.cpp

```diff
@@ -1404,23 +1395,33 @@ void ObjectStopAnim(Object &object)
 	}
 }
 
+/**
+ * @brief Checks if an open door can be closed
+ *
+ * In order to be able to close a door the space where the closed door would be must be free of bodies, monsters, and items
+ *
+ * @param doorPosition Map tile where the door is in its closed position
+ * @return true if the door is free to be closed, false if anything is blocking it
+ */
+inline bool IsDoorClear(const Point &doorPosition)

```

Comment:
Where possible I've been trying to use constexpr over inline (doesn't make much difference, just acts as a stronger suggestion afaik), as long as the eventual `IsMonsterAtPosition` etc functions are also made constexpr this should be able to use that keyword.
```suggestion
constexpr bool IsDoorClear(const Point &doorPosition)
```

Reply:
ill include it in the follow up as i have to touch this any way

Reply:
turns out this doesn't build, so it's still `inline` in the other PR :)